src/full/Agda/Syntax/Internal.hs

```diff
@@ -367,29 +367,29 @@ type NAPs = [NamedArg DeBruijnPattern]
 --  For the purpose of the permutation and the body dot patterns count
 --  as variables. TODO: Change this!
 data Clause = Clause
-    { clauseLHSRange  :: Range
+    { clauseLHSRange :: Range
```

Comment:
Why remove the alignment?
The next PR might introduce the alignment again...
Also, in other places, you introduce alignment.
I think we should just go in one direction.
Either leave whitespace alone, or maybe introduce alignment.

Reply:
In this case, alignment seemed redundant given the comment sizes below each field. Since every vertical alignment also introduces a maintanability burden, I personally only introduce it when it makes code easier to read, so I opted to remove it here. I'd be happy to revert this though to agree with the general style Agda uses, WDYT?

Reply:
In my experience, folks
- introduce alignment
- break alignment when introducing new cases
- sometimes restore alignment
- sometimes remove alignment for catch-all clauses

I think all of these are fine.  But actively removing alignment usually does not happen.  If in doubt, leave the whitespace alone (reduces potential for merge conflicts).

I don't feel strongly in this particular case, though: -0.5.

Reply:
P.S.: That we even discuss this, is a shortcoming of the PL mainstream.  Our PLs should really support partial alignment and tabular layouts natively.

Reply:
> I don't feel strongly in this particular case, though: -0.5.

Now reverted.